src/Translator/src/DependencyInjection/TranslatorCompilerPass.php

```diff
@@ -25,6 +34,7 @@ private function hasValidTranslator(ContainerBuilder $containerBuilder): bool
         if (!is_subclass_of($translator->getClass(), TranslatorBagInterface::class)) {
             return false;
         }
+
         return true;
     }
-}
\ No newline at end of file
+}
```

Comment:
Maybe revert this one to not bring this (amazing) PR on other componenets ?

Reply:
You're right, just opened #1996